core/templates/pages/exploration-player-page/templates/lesson-information-card-modal.component.ts

```diff
@@ -276,6 +282,37 @@ export class LessonInformationCardModalComponent extends ConfirmOrCancelModal {
     });
   }
 
+  getCheckpointsCardIndexs(): number[] {
```

Comment:
I think it's probably worth adding JSDoc to explain what this returns, the return type / value, etc. Ditto for the method below.

Reply:
Done